bin/python-noenv.sh

```diff
@@ -1,2 +1,2 @@
 #!/bin/bash
-env -i /usr/bin/python "$@"
+env -i PBS_DEFAULT=$PBS_DEFAULT /usr/bin/python "$@"
```

Comment:
hold on, what do you mean with "minimal combo" exactly?

I figured that was using `python -E` + unsetting all `LD_*` environment variables?